config/pacbio_instrument_types.yml

```diff
@@ -16,7 +16,12 @@ default: &default
               attribute: :plates
               minimum: 1
               maximum: 1
-          # TODO: fix this to have separate behaviour for v11 and v12 (see commit 05bfe764a8eafe)
+          required_attributes:
+            options:
+              versions:
```

Comment:
We can definitely remove the versions code but wait to hear back from Craig as to whether this needs to be required. It wasn't previously.